src/gaps/QueryFilterParser.ts

```diff
@@ -377,7 +382,7 @@ export function interpretFunctionRef(functionRef: ELMFunctionRef, library: ELM):
           break;
       }
     } else {
-      console.warn(`do not know how to interpret function ref ${functionRef.libraryName}."${functionRef.name}"`);
+      //(`do not know how to interpret function ref ${functionRef.libraryName}."${functionRef.name}"`);
```

Comment:
was this commented out line supposed to be another withError aggregation that was missed?